docs-site/src/content/docs/fr/reference/cli/lifecycle.md

```
@@ -0,0 +1,272 @@
+
+```bash
+ocx service
+ocx service install
+ocx service repair
+ocx service status
+ocx service uninstall
+```
+
+Avant de signaler une réussite, `install`, `start` et `repair` vérifient, sur les trois plateformes, qu’un proxy répond effectivement sur le port inscrit dans le service installé. Elles attendent jusqu’à 20 secondes, puis affichent le port utilisé :
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)